benchmarks/serverRouteDispatch.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+var Brisket = require("../lib/brisket");
+var RouterBrewery = Brisket.RouterBrewery;
+var ServerDispatcher = require("../lib/server/ServerDispatcher");
+
+Brisket.Testing.setup();
+
+var Router = RouterBrewery.create({
+
+    layout: Brisket.Layout,
+
+    routes: {
+        "path": "handler"
+    },
+
+    handler: function() {
+        return new Brisket.View();
+    }
+
+});
+
+new Router();
```

Comment:
when you initialize a backbone router, it's routes are added to the global dispatcher. on server side, it's ServerDispatcher and on the client it's Backbone.History